cli/src/pcluster/resources/custom_resources/custom_resources_code/crhelper/utils.py

```diff
@@ -17,17 +17,17 @@ def _send_response(response_url, response_body):
     except Exception as e:
         msg = "Failed to convert response to json: {}".format(str(e))
         logger.error(msg, exc_info=True)
-        response_body = {'Status': 'FAILED', 'Data': {}, 'Reason': msg}
+        response_body = {"Status": "FAILED", "Data": {}, "Reason": msg}
```

Comment:
I was keeping the formatting checks on this file disabled since we are importing it from https://github.com/aws-cloudformation/custom-resource-helper.

Reply:
Is it an issue to make the changes here? That enables it to play more nicely with pre-commit.